web-console/4.0-designs/user/user.md

```diff
@@ -0,0 +1,19 @@
+# User Menu
+
+## Copy Login Command
+![pic 0](img/copy-login-command-1.png)
```

Comment:
@beanh66 small nit, but can you change Logout to "Log out" ?  I know that Robb was putting that through this week (applies to all 3 images)